js/tabs.js

```diff
@@ -40,18 +40,30 @@ function setupToolsTabs(container, tabIdPrefix, storageName) {
       history.replaceState(undefined, undefined, '#' + selectedTool);
 
     // Persist in local storage so we can pre-select around the site
-    if (window.localStorage)
+    if (storageName && window.localStorage)
       window.localStorage.setItem(storageName, selectedTool);
   });
 
   // If we have a tool in the url fragement, pre-select it
   if (location.hash && location.hash.length > 1)
     selectTool(location.hash.substr(1));
-  else if (window.localStorage && window.localStorage.getItem(storageName))
+  else if (storageName && window.localStorage && window.localStorage.getItem(storageName))
     selectTool(window.localStorage.getItem(storageName));
+  else if (defaultTab)
+    selectTool(defaultTab);
 }
 
 $(document).ready(function () {
   setupToolsTabs($("#tab-set-install"), "tab-install-", "selectedTool");
-  setupToolsTabs($("#tab-set-os"), "tab-os-", "selectedOS");
+  setupToolsTabs($("#tab-set-os"), "tab-os-", null, getOS());
 });
+
+function getOS() {
+  var ua = navigator.userAgent.toLowerCase();
+  if (ua.indexOf("win") !== -1)
+    return "windows";
+  if (ua.indexOf("mac") !== -1)
+    return "macos";
+  if (ua.indexOf("linux") !== -1)
+    return "linux";
```

Comment:
The only Unixes we officially support are Linux and macOS, but you could add a test for "X11" to the Linux test to catch any other types of Unix. Even if they're not running on the machine they're downloading it on, someone running on a non-Linux, non-Mac, Unix browser will probably want the Linux package.

Also, it might be better to compare in a case-sensitive way ("Mac", "Win", "Linux"), since "win" and "mac" are really short strings, and might appear in other strings (e.g. "Darwin").

Reply:
Done (and re-staged). It's possible we could could expand Win -> Windows and Mac -> Macintosh but I don't know if there's much value now they're case sensitive (or how consistent that is across browsers).

Reply:
No, no need to expand those, IMO, and they might even catch some edge cases if you don't.